src/storage-client/src/client.rs

```diff
@@ -468,6 +520,12 @@ where
                     Some(Ok(StorageResponse::DroppedIds(new_drops)))
                 }
             }
+            StorageResponse::StatisticsUpdates(stats) => {
+                // Just forward it along; the `worker_id` should have been set in `storage_state`.
```

Comment:
Note to self: talk to Gus about the reasoning for this

Reply:
hello!